mapping-for-navigation/_posts/0000-01-10-circular-junctions.md

```diff
@@ -0,0 +1,208 @@
+
+[1108080d]: https://wiki.openstreetmap.org/wiki/Key:highway
+
+[e1236adf]: https://wiki.openstreetmap.org/wiki/Tag:highway%3Dturning_loop
+
+[496a5a09]: https://wiki.openstreetmap.org/wiki/Tag:highway%3Dturning_circle
+
+[bca8053e]: https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout
+
+[f14df547]: https://wiki.openstreetmap.org/wiki/Key:junction
+
+See examples of mini-roundabouts in the [OpenStreetMap Wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dmini_roundabout#Mini-roundabouts).
+
+### Roundabout
+
+A [roundabout](https://en.wikipedia.org/wiki/Roundabout) is a generally circular road [junction](https://wiki.openstreetmap.org/wiki/Junctions) where the traffic on the roundabout has right of way and tagged with **[junction][f14df547]**=**[roundabout](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout)** tag. This tag is used only on road intersections where traffic on the roundabout has right of way as in the absence of traffic signals in these intersections.
+
+[![roundabout][108eb915]][108eb915]
+<sup>_Take a look at a [wireframe view][c134f465] of this roundabout to get some details from the imagery_</sup>
+
+  [108eb915]: ../../images/intersection-modeling/roundabouts/roundabout.png "roundabout"
+  [c134f465]: ../../images/intersection-modeling/roundabouts/roundabout.png "wireframe view"
+
+**How to map**
+
+To indicate a roundabout — draw a closed line in the direction the traffic flows , the same as you do it during the road tracing and tag it with the **[junction][f14df547]**=**[roundabout](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout)** tag and appropriate **`highway=*`** tag value.
+
+Roundabout is implied to be one way road but you may specify this explicitly by adding **[oneway](https://wiki.openstreetmap.org/wiki/Key:oneway)**=**[yes](https://wiki.openstreetmap.org/wiki/Tag:oneway%3Dyes)** tag. Additionally the number of **[lanes](https://wiki.openstreetmap.org/wiki/Key:lanes)**=**`*`** in the circle (typically 2, where long vehicles will need to use both) can also be tagged.
+
+Connect road lines from which the traffic flows to the roundabout. If a dual carriageway shares traffic with the roundabout, connect each its direction (one approaching, one departing) separately to the respective exit and entrance nodes on the roundabout. Don’t use the same node as entrance to and exit from a roundabout. Each road has to be connected with the roundabout in a separate node.
+
+When two roads share the same exit/entrance node connect them before the roundabout.
+
+[![][8c17feb2]][8c17feb2]
+
+  [8c17feb2]: ../../images/intersection-modeling/roundabouts/roundabout_pinch_ways.png
+
+**Roundabout shapes**
+
+Roundabouts are not necessarily circular in shape: there does exist roundabouts that are 0-shaped (oval) or D-shaped (with a straight segment, notably those created on a square whose one side is along an avenue).
```

Comment:
there does exist roundabouts that --> there are roundabouts that

just slightly less wordy